src/main/kotlin/lotto/LottoGame.kt

```diff
@@ -0,0 +1,26 @@
+package lotto
+
+import WinningResult
+import lotto.LottoConstants.LOTTO_PRICE
+
+class LottoGame {
+    fun purchase(paidMoney: Int): List<Lotto> {
+        val purchaseTime = paidMoney / LOTTO_PRICE
+        require(paidMoney >= LOTTO_PRICE) { "로또 구입 금액은 1000원 이상이어야 합니다." }
+
+        return (1.rangeTo(purchaseTime)).map { LottoRandomGenerator().randomGenerate() }
```

Comment:
저는 이런 표현이 조금 더 가독성이 좋다고 생각해서 가능하면 적극적으로 활용하는 편이에요. 😃

```suggestion
        return (1..purchaseTime).map { LottoRandomGenerator().randomGenerate() }
```